gravitropism.py

```diff
@@ -0,0 +1,38 @@
+import math
+
+def calculate_results_gravitropism(x1, y1, x2, y2, time_elapsed):
+    """
+    Calculate distance, rate, and angle of movement.
+
+    Parameters:
+        x1, y1: initial coordinates of the plant
+        x2, y2: final coordinates of the plant
+        time_elapsed: time elapsed between the two points
+
+    Returns:
+    dictionary of 
+        distance: distance moved (mm)
+        rate: rate of movement (mm/min)
+        angle: angle of movement in degrees relative to positive x-axis
+    """
```

Comment:
This is a great start! I suggest you adopt the python sphinx docstrings standard for specifying the parameters and returns; you can find out about it here:

https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
yes, let's do that